Readme.md

```diff
@@ -101,7 +101,7 @@ To work this around:
 ## Usage
 
 - [Laravel GraphQL](#laravel-graphql)
-    - [Note: these are the docs for 3.*, please see the `v1` branch for the 1.* docs](#note-these-are-the-docs-for-3-please-see-the-v1-branch-for-the-1-docs)
+    - [Note: these are the docs for the current release, please see the `v1` branch for the 1.* docs](#note-these-are-the-docs-for-the-current-release-please-see-the-v1-branch-for-the-1-docs)
```

Comment:
How is this diff even possible? The master branch literally says this: https://github.com/rebing/graphql-laravel/commit/30e69d32ada916ae5f7caeca4783c0380f167aa7

Reply:
probably my markdown editor that changed that, will revert

Reply:
No, I mean the version in master already is at the change so technically this shouldn't be possible.

No need to change it, it's the _correct_ text anyway!